third_party/java/arcs/build_defs/internal/kotlin.bzl

```diff
@@ -74,7 +74,8 @@ def arcs_kt_jvm_library(**kwargs):
     disable_lint_checks = kwargs.pop("disable_lint_checks", [])
     exports = kwargs.pop("exports", [])
     kotlincopts = kwargs.pop("kotlincopts", [])
-    kwargs["kotlincopts"] = merge_lists(kotlincopts, KOTLINC_OPTS)
+    # jvm-target 1.8 needed to solve crash in Bazel Desugaring
+    kwargs["kotlincopts"] = merge_lists(kotlincopts, KOTLINC_OPTS + ["-jvm-target 1.8"])
```

Comment:
Is the issue just for bazel or for blaze too?

Assuming just for bazel, I'd suggest defining a new `BAZEL_KOTLINC_OPTS` variable with this flag, and then adding a `else` branch to the if-statement below, and doing the `merge_lists` there.

Reply:
I'll test it, and if it is BAZEL only, I'll make the change.